mysql-test/suite/rocksdb/t/fast_prefix_index_fetch.test

```diff
@@ -0,0 +1,97 @@
+--source include/have_rocksdb.inc
```

Comment:
Thanks for the heads up. I used your testcase to clean up and add tests for utf8. I don't think we are affected by this issue but it's good to know.